OpenStack/RHEL-OSP/7/CMakeLists.txt

```diff
@@ -1 +1,8 @@
 ssg_build_product("rhel-osp7")
+
+if (SSG_CENTOS_DERIVATIVES_ENABLED)
+    ssg_build_derivative_product("rhel-osp7" "centos" "centos-osp7")
+endif()
+if (SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED)
+    ssg_build_derivative_product("rhel-osp7" "sl" "sl-osp7")
+endif()
```

Comment:
Are we sure that there is a CentOS and SL OSP? I get the feeling that there is not, but couldn't find any information that says either way.